DataCatalog/examples/0.3-RELEASE/MobiDB_jsonld.json

```diff
@@ -0,0 +1,98 @@
+{
+  "@context": "http://schema.org/",
+  "@type": "DataCatalog",
+  "@id": "https://mobidb.org/",
+  "dct:conformsTo": "https://bioschemas.org/profiles/DataCatalog/0.3-RELEASE-2019_07_01",
+  "sameAs":"https://registry.identifiers.org/registry/mobidb",
+  "url": "https://mobidb.org/",
+  "identifier": "https://registry.identifiers.org/registry/mobidb",
+  "name": "MobiDB, a database of protein disorder and mobility annotations",
+  "description": "MobiDB is a database of protein disorder and mobility annotations. MobiDB was designed to offer a centralized resource for annotations of intrinsic protein disorder and its function. The database covers different disorder aspects such as disordered regions lacking a fixed three-dimensional structure, linear interacting peptides and dynamic structures",
+  "datePublished": "2019-09",
+  "dateModified": "2020-03",
+  "citation": {
+    "@type": "ScholarlyArticle",
+    "@id": "https://doi.org/10.1093/nar/gkx1071",
+    "name": "MobiDB 3.0: more annotations for intrinsic disorder, conformational diversity and interactions in proteins",
+    "url": "https://doi.org/10.1093/nar/gkx1071",
+    "sameAs": [
+      "https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkx1071",
+      "https://www.ncbi.nlm.nih.gov/pubmed/29136219"
+    ]
+  },
+  "keywords": [
```

Comment:
Keywords should be a single string where actual keywords are separated by commas